### PR TITLE
#312 update service tests for inbound invoice

### LINKED ISSUE #312: Update service tests for inbound invoice

### PR DESCRIPTION
**Pull request by roxy-dao**
Closes #312

### CODE REVIEW

File: server/service/src/invoice/inbound_shipment/update/mod.rs [hunk 1]
```diff
@@ -209,6 +216,70 @@ mod test {
...
+                &mock_store_a().id,
+                "n/a",
+                inline_init(|r: &mut UpdateInboundShipment| {
+                    r.id = mock_outbound_shipment_c().id.clone();
+                    r.other_party_id = Some(mock_name_a().id.clone());
+                })
+            ),
+            Err(ServiceError::NotAnInboundShipment)
+        );
+        //NotThisStoreInvoice
```
Comment (roxy-dao) [inline server/service/src/invoice/inbound_shipment/update/mod.rs:245]: Can still update invoice even if the store_id doesn't match 🤔

File: server/service/src/invoice/mod.rs [hunk 2]
```diff
@@ -81,7 +81,7 @@ pub trait InvoiceServiceTrait: Sync + Send {
         user_id: &str,
         input: DeleteInboundShipment,
     ) -> Result<String, DeleteInboundShipmentError> {
-        delete_inbound_shipment(ctx, user_id, store_id, input)
+        delete_inbound_shipment(ctx, store_id, user_id, input)
```
Comment (roxy-dao) [inline server/service/src/invoice/mod.rs:84]: Parameters wrong way around.

File: server/service/src/invoice/inbound_shipment/update/mod.rs (hunk 1 above)
Comment (mark-prins) [reply server/service/src/invoice/inbound_shipment/update/mod.rs:245]: I think create a new issue and add it in. not sure why it isn't being checked

**Review by mark-prins — APPROVED** [review]
👍  LGTM!

**Pull request merged**